.travis.yml

```diff
@@ -11,52 +11,20 @@ env:
 
 matrix:
   include:
-    - php: 5.6
-      env:
-        - DEPS=lowest
-    - php: 5.6
-      env:
-        - DEPS=locked
-        - LEGACY_DEPS="phpunit/phpunit"
-    - php: 5.6
-      env:
-        - DEPS=latest
-    - php: 7
-      env:
-        - DEPS=lowest
-    - php: 7
-      env:
-        - DEPS=locked
-        - LEGACY_DEPS="phpunit/phpunit"
-    - php: 7
-      env:
-        - DEPS=latest
-    - php: 7.1
-      env:
-        - DEPS=lowest
-    - php: 7.1
-      env:
-        - DEPS=locked
-        - CS_CHECK=true
-        - TEST_COVERAGE=true
-    - php: 7.1
-      env:
-        - DEPS=latest
     - php: 7.2
       env:
         - DEPS=lowest
     - php: 7.2
       env:
         - DEPS=locked
+        - CS_CHECK=true
+        - TEST_COVERAGE=true
     - php: 7.2
       env:
         - DEPS=latest
     - php: 7.3
       env:
         - DEPS=lowest
-    - php: 7.3
-      env:
-        - DEPS=locked
```

Comment:
Locked dependencies are there to tell us when breakage comes from dependency and not the change in PR. Having lock file for a single version is sufficient for that, since it is made for specific version and it avoids errors like we had with Doctrine instantiator all over the framework because of lock file on wrong version.

Reply:
Still we can have this issue, if someone regenerate new lock file on PHP 7.3. But I see your point. Just thinking maybe it will be better to run on the "latest" stable, supported PHP version then?

Reply:
If packages are incompatible, PR will error out. A fix would be to update packages on different version. No legacy deps workarounds in build config.

Makes sense. Then cs check and coverage needs to be moved to latest too.